doc/api/http.md

```diff
@@ -556,7 +556,7 @@ Returns `request`.
 added: v0.1.17
 -->
 
-This class inherits from [`net.Server`][] and has the following additional events:
+This�class inherits from [`net.Server`][] and has the following additional events:
```

Comment:
Just to be sure, the extra whitespace at the end of this line has been removed, right?